README.md

```diff
@@ -151,6 +153,30 @@ pip install -e ".[gpu]"  # or pip install -e . if no NVIDIA GPU
 ```
 
 
+### AMD (BETA support)
+
+In [our testing of AMD GPUs](https://www.mosaicml.com/blog/amd-mi250), the env setup includes:
+
+<!--pytest.mark.skip-->
+```bash
+git clone https://github.com/mosaicml/llm-foundry.git
+cd llm-foundry
+
+# Creating and activate a virtual environment
+python3 -m venv llmfoundry-venv-amd
+source llmfoundry-venv-amd/bin/activate
+
+# installs
+pip install cmake packaging torch
```

Comment:
should torch be here?

Reply:
yes
its a bit screwy

torch is needed to install some deps
then out `pip install -e .` installs whatever specified version of torch it decides (usually a CPU version)
then we have to reinstall the amd torch version

Reply:
actually let me try without that torch
maybe we just need to install torch twice instead of 3 times...

Reply:
ok so it works IFF the system already has some version of torch installed.
in case ppl don't have that going for them, these instruction are complete.
worse case, pip says: requirement satisfied.